src/App.js

```diff
@@ -1,122 +1,99 @@
-                </button>
-            </div>
-            <div className="display-section">
-                {!isChosen ? (
-                    <h1>Please choose a pokemon</h1>
-                ) : (
-                    <div className="stats-window">
-                        <h1 className="stats-title">{pokemonInfo.name}</h1>
-                        <div className="text-and-image-wrapper">
-                            <div className="text-stats">
-                                <p>
-                                    <span className="bold">Species:</span>{" "}
-                                    {pokemonInfo.species}
-                                </p>
-                                <p>
-                                    <span className="bold">Type:</span>{" "}
-                                    {pokemonInfo.type}
-                                </p>
-                                <p>
-                                    <span className="bold">HP:</span>{" "}
-                                    {pokemonInfo.hp}
-                                </p>
-                                <p>
-                                    <span className="bold">Attack:</span>{" "}
-                                    {pokemonInfo.attack}
-                                </p>
-                                <p>
-                                    <span className="bold">Defence:</span>{" "}
-                                    {pokemonInfo.defense}
-                                </p>
-                            </div>
-                            <div className="image-window">
-                                <img
-                                    className="pokemon-image"
-                                    src={pokemonInfo.imageUrl}
-                                    alt="front view of chosen pokemon"
-                                ></img>
-                            </div>
-                        </div>
-                    </div>
-                )}
-            </div>
-        </div>
-    );
+  }, []);
+
+  const handleChosen = () => {
+    if (pokemonName) {
+      setIsChosen(true)
+    }
+  }
+
+  useEffect(() => {
+    if (location.pathname !== "/") {
+      setIsChosen(true)
+    } else {
+      setIsChosen(false);
+    }
+  }, [location.pathname]);
+
+  return (
+    <div className="App">
+      <div className="title-section">
+        <h1 className="title">Poke Stats</h1>
+        <input
+          className="search-input"
+          type="text"
+          onChange={(event) => {
+            setPokemonName(event.target.value);
+          }}
+        ></input>
+        <Link to={{ pathname: `/${pokemonName}`}}>
+          <button id="search-button" className="search-button" onClick={handleChosen}>
+            Search Pokemon
+          </button>
+        </Link>
+      </div>
+      {isChosen ? (
+        <Outlet />
+      ) : (
+        <h1 className="call-to-action">Please Choose a Pokemon</h1>
+      )}
```

Comment:
This conditional rendering of the `Outlet` is surprising imo (and non-idiomatic for react router)

This should _always_ have `Outlet` rendered and then your Route configuration below should decide _what_ that `Outlet` actually shows. You can make use of [index routes](https://reactrouter.com/en/main/route/route#index) to manage what's rendered when there's no other path segments.

Reply:
We talked through and changed this. The Outlet is now rendered by React Router and shows conditionally without me controlling it with state.